tpgtools/product.go

```diff
@@ -93,6 +90,31 @@ func (pm *ProductMetadata) ProductBasePathDetails() *ProductBasePathDetails {
 	return &bp
 }
 
+func getProductTitle(documentTitle, packagePath string) string {
+	overrides, ok := productOverrides[packagePath]
+	if !ok {
+		glog.Fatalf("product overrides should be loaded already for packagePath %s", packagePath)
+	}
+
+	bt := ProductTitleDetails{}
+	btOk, err := overrides.ProductOverrideWithDetails(ProductTitle, &bt)
+	if err != nil {
+		log.Fatalln("error - failed to decode base path details")
+	}
+
+	if btOk && bt.Title != "" {
```

Comment:
Can we return an error when `bt.Title == ""` instead of silently failing? A panic is fine if amending the signature(s) of callers is too hard.